tests/e2e/support/objects/app-admin-settings/users/actions.ts

```diff
@@ -14,7 +14,7 @@ const dropdownOption = '.vs__dropdown-option'
 const loginInput = '#login-input'
 const compareDialogConfirm = '.compare-save-dialog-confirm-btn'
 const addToGroupsBatchAction = '.oc-users-actions-add-to-groups-trigger'
-const removeFromGroupsBatchAction = '.oc-users-actions-remove-from-groups-trigger'
+const removeFromGroupsBatchAction = '.oc-users-actions-edit-quota-trigger'
```

Comment:
Huh, something got mixed up here, no? Either the var name or the selector is wrong.

Reply:
wtf